lib/utils/transform.py

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env python3
-import regex
-import pandas as pd
 from datetime import datetime
 from typing import Optional, Set, Union
 
+import pandas as pd
+import regex
+
```

Comment:
Unnecessary diff

Reply:
Comes from sorted imports

Reply:
I understand, but we aren't enforcing that on any other files in this repo. I'd consider this change in another PR that applies sorted imports to all files at once.

Reply:
An argument against bundling this change in c69e0bf064f8f8e0f588ada475ee846ef64d40aa is that it is confusing to anyone searching through `git blame` to understand file history:

![image](https://user-images.githubusercontent.com/13424970/204624412-132cbf29-726d-4d46-b9b4-4a49fd9d554d.png)

Without digging further, this indicates that the change adding `genbank_data_rev` uses `pandas` and `regex` in some fashion, when really it doesn't.

Reply:
Yep you're right! The tricky thing is: I want new files to start off correctly so I'd have to set formatting on a file by file basis.

I know @tsibley isn't a fan of code formatters. I guess we'd need to agree and enforce it otherwise things deteriorate.